python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, jobDirectory='', registry=None, dataFileName='data', load_bac
             return
 
         self._subjobIndexData = {}
+        if parent is not None:
+            self._setParent(parent)
```

Comment:
Just to be sure, this is only changing the parent if it already has one and is leaving it as `None` if it was before?